modules/deploy/pages/deployment-option/cloud/whats-new-cloud.adoc

```diff
@@ -5,6 +5,15 @@
 
 This page lists new features added in Redpanda Cloud.
 
+== July 2024
+
+=== Support for additional regions
+
+For xref:deploy:deployment-option/cloud/byoc-tiers.adoc#byoc-supported-regions[BYOC clusters] and xref:deploy:deployment-option/cloud/dedicated/dedicated-tiers.adoc#dedicated-supported-regions[Dedicated clusters], Redpanda added support for the following regions:
+
+* GPC: asia-east1 (UAE), asia-northeast1 (Tokyo), southamerica-east1 (São Paulo)
```

Comment:
This incorrectly says UAE. This should be Taiwan as per the Aha https://redpanda.aha.io/features/CD-173